arch/arm/dts/tegra186-p2771-0000.dtsi

```diff
@@ -12,6 +12,14 @@
 		sdhci0 = "/sdhci@3460000";
 		sdhci1 = "/sdhci@3400000";
 		i2c0 = "/bpmp-i2c";
+		i2c1 = "/i2c@3160000";
```

Comment:
Are you sure all of these I2C buses are actually used on this board? Did you test them? It might be a good idea only to enable a minimal set of buses that are known to contain useful devices.